### PR TITLE
REFS-initial version 

### PR DESCRIPTION
**Pull request by BinbinZhou-NOAA**
## Pull Request Testing ##

Note 1:The testing of REFS components should be done for both EVSv1(MET/11.0.x METplus/5.0.x) and EVSv2 (MET/12.0.x, METplus/v6.0.x). The current run.ver and modulefiles are for EVSv2. If testing for EVSv1, they should be changed accordingly.
 
Note 2: The input REFS forecasting files are from the REFS output files run by Matt Pyle in his personal account. So there may be missing files. But this will not stop the testing except for the case in which more than 50%  of the member files are missing. 

Note 3: For the plotting job testing:  The COMIN should be pointed to /lfs/h2/emc/vpppg/noscrub/binbin.zhou/evs/v1.0/stats
for EVSv1 and to /lfs/h2/emc/vpppg/noscrub/binbin.zhou/evs/v2.0/stats for EVSv2, respectively.

Note 4. since the number of days of stat files is less than 90 days. In other words, there are about 
60days of stat files. So for 90day plotting jobs, just those 60 days of stat files are available. But the tar file names are still marked as 90 days.

Note5. The METplus 5 do not work for the ECNT stats generated from  METplus/v6   
But METplus/v6 can work for the stats generated from METplus/v5 
So in testing of the ecnt and profile plotting jobs for EVSv1, the stat files generated from METplus/6 must be excluded. 
In other words, removed all stat directories before 20240623 in the COMIN. But since COMIN is pointed to my personal account, this is not possible. The best away is set the export past_days=3 (or 4, or 5, ....,  depending on the date) in the ecnt and profile job driver scripts.  In this case, there are no full 31 or 90 days stat files.  Right now I am routinely run REFS in METplus/v5. So 
after 20240623, all of the stat files in my account (COMIN) are generated from METplus/v5 that can be used by both
EVSv1 and EVSv2 testing. 

The general testing procedures:
     Similar to HREF testing: first run 3 stats generation driver scripts. The 3 stats jobs have no
dependency. Then run the driver scripts for all plotting jobs. They have no dependency either.

### CODE REVIEW

**Comment by ShelleyMelchior-NOAA** [comment]
@BinbinZhou-NOAA , thank you for your effort on this PR! There will be some updates to follow next week!

**Comment by BinbinZhou-NOAA** [comment]
This PR should be off the feature/rrfs-refs-v1 instead of develop. So just close this PR.
